charts/keycloak-operator/values.yaml

```diff
@@ -0,0 +1,283 @@
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  # -- Node selector for Operator Deployment.
+  nodeSelector: {}
+
+  # -- Tolerations for Operator Deployment.
+  tolerations: []
+
+  # -- Affinity for Operator Deployment.
+  affinity: {}
+
+keycloak:
+  # -- Enable deploying a bundled keycloak with the operator
+  enabled: false
+
+  # -- Number of Keycloak instances in HA mode.
+  instances: 1
+
+  transaction:
+    # -- Determine whether Keycloak should use a non-XA datasource.
+    xaEnabled: false
+
+  http:
+    # -- The used HTTP port
+    httpPort: ~
+    # -- A secret containing the TLS configuration for HTTPS.
+    tlsSecret: ~
+    # -- The used HTTPS port
+    httpsPort: ~
+    # -- Enable a HTTP listener
+    httpEnabled: true
+
+  hostname:
+    # -- Hostname for the Keycloak server.
+    # @default -- Disabled if not set.
+    hostname: ~
+    # -- Disables dynamically resolving the hostname from request headers
+    strict: false
+    # -- By default backchannel URLs are dynamically resolved from request headers to allow internal and external applications.
+    strictBackchannel: true
+    # -- The hostname for accessing the administration console.
+    admin: ~
+    # -- Set the base URL for accessing the administration console.
+    adminUrl: ~
+
+  unsupported:
+    # -- You can configure values that will be merged with the one configured by default by the operator.
+    podTemplate: ~
+      #    metadata:
+      #      labels:
+      #        foo: "bar"
+      #    spec:
+      #      containers:
+      #        - volumeMounts:
+      #            - name: test-volume
+      #              mountPath: /mnt/test
+      #          command: [ "/bin/bash", "-c", "cat /mnt/test/test.txt && /opt/keycloak/bin/kc.sh start-dev" ]
+      #      volumes:
+      #        - name: test-volume
+      #          secret:
+      #            secretName: keycloak-podtemplate-secret
+
+  ingress:
+    # -- The deployment is, by default, exposed through a basic ingress.
+    enabled: true
+
+  image:
+    # -- Overrides the operator.keycloakImage.image value whose default is quay.io/keycloak/keycloak
+    repository: ""
+    # -- Overrides the operator.keycloakImage.tag value whose default is the chart appVersion.
+    tag: ""
+
+  # -- Secret(s) that might be used when pulling an image from a private container image registry or repository.
+  imagePullSecrets: ~
+  # - name: secret-name
+
+  # -- Configuration of the Keycloak server expressed as a keys and values that can be either direct values or references to secrets.
+  additionalOptions: ~
+  # - name: string
+  #   value: string
+  #   secret:
+  #     optional: bol
+  #     key: string
+  #     name: string
+
+  db:
```

Comment:
What's your opinion regarding adding a database chart as a dependency? It would make local development easier, but on the other hand some people might use in in production without knowing what they're getting themselves into.

Reply:
I've been thinking about that as well, but decided to not include a DB chart in the 0.1.0 release